firefox-ios/WidgetKit/OpenTabs/OpenTabsWidget.swift

```diff
@@ -81,7 +81,7 @@ struct OpenTabsView: View {
                     Text(String.NoOpenTabsLabel)
                     HStack {
                         Spacer()
-                        Image("externalLinkSmall")
+                        Image(decorative: "externalLinkSmall")
```

Comment:
nit: not part of this PR since pre-existing, but should these hardcoded image strings live elsewhere? also seems like its duplicate, so updating this can have a single place where it's updated.

Reply:
Good point. I think we had them hardcoded in the past because the widgets and extensions didn't include `Common` and also they have their own asset catalogs (I think because of size restrictions for extensions). I'll create a task to use `StandardImageIdentifiers` here as well.